Skype/WebSDK/docs/PTVideoOutgoingP2P.md

```diff
@@ -23,32 +26,29 @@ After the conversation and video modality are established we can begin communica
 1. Initiate a video conversation with a person, and set up associated listeners 
 
   ```js
-    var id = content.querySelector('.id').value;
     var conversationsManager = application.conversationsManager;
-    conversation = conversationsManager.getConversation(id);
-
-    function setupContainer(person, size, videoDiv) {
-        person.video.channels(0).stream.source.sink.format('Stretch');
-        person.video.channels(0).stream.source.sink.container(videoDiv);
-    }
+    conversation = conversationsManager.getConversation('sip:xxx');
 
-    listeners.push(conversation.selfParticipant.video.state.when('Connected', function () {
+    conversation.selfParticipant.video.state.when('Connected', function () {
         // set up self video container
-        setupContainer(conversation.selfParticipant, 'small', content.querySelector('.selfVideoContainer'));
-    }));
-    listeners.push(conversation.participants.added(function (person) {
+        conversation.selfParticipant.video.channels(0).stream.source.sink.format('Stretch');
+        conversation.selfParticipant.video.channels(0).stream.source.sink.container(/* DOM node such as DIV */);
+    });
+    conversation.participants.added(function (person) {
         console.log(person.displayName() + ' has joined the conversation');
```

Comment:
usually this is just a comment (// person.displayName() + ' has joined the conversation') - we should be consistent